src/components/Pokemon.js

```diff
@@ -0,0 +1,28 @@
+const Pokemon = ({ pokemonType, userSelection }) => {
+  const slicedPokemonType = pokemonType.slice(0, 10)
```

Comment:
This has been a lot of fun to read through so far. Your code is so well organized that I can envision the app without seeing the app. Great call using .slice to determine a pokemonType.